build-tools/scripts/RunNUnitTests.targets

```diff
@@ -33,7 +33,7 @@
     <SetEnvironmentVariable Name="JAVA_INTEROP_LREF_LOG" Value="bin\Test$(Configuration)\l-%(_TestAssembly.Filename).txt" />
     <SetEnvironmentVariable Name="JI_JVM_PATH"           Value="$(JdkJvmPath)" />
     <Exec
-        Command="$(_NUnit) $(NUNIT_EXTRA) %(_TestAssembly.Identity) $(_Run) --result=&quot;TestResult-%(Filename).xml;format=nunit2&quot; --output=&quot;bin\Test$(Configuration)\TestOutput-%(Filename).txt&quot;"
+        Command="$(_NUnit) $(NUNIT_EXTRA) %(_TestAssembly.Identity) $(_Run) --result=&quot;TestResult-%(Filename).xml&quot; --output=&quot;bin\Test$(Configuration)\TestOutput-%(Filename).txt&quot;"
```

Comment:
Why is `format=nunit2` not needed?  Does AzDO recognize the default format?  Did the default change?

Reply:
I'm not sure why, but when I left it in there, the CI NUnit choked on it and said it wasn't a valid format.  Removing it seems to work, the tests are still picked up for the test tab:

https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3266847&view=ms.vss-test-web.build-test-results-tab

I think now it defaults to `nunit3`.  Given that NUnit 3.0 shipped in 2015, I'm more surprised that AzDO supported `nunit2`.  ;)